src/test/scala-2.11/markets/participants/TestOrderCanceler.scala

```diff
@@ -27,13 +28,14 @@ import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.ExecutionContext.Implicits.global
 
 
-class TestOrderCanceler(initialDelay: FiniteDuration,
-                        markets: mutable.Map[Tradable, ActorRef],
-                        tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
-  extends TestMarketParticipant(markets, tickers)
-  with OrderCanceler[TestOrderCancellationStrategy] {
+case class TestOrderCanceler(initialDelay: FiniteDuration,
+                             markets: mutable.Map[Tradable, ActorRef],
+                             tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
+  extends OrderCanceler[TestOrderCancellationStrategy] {
 
-  orderPlacementStrategy.scheduleOnce(initialDelay, self, SubmitOrderCancellation)
+  val outstandingOrders = mutable.Set.empty[Order]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2091096922/issues/source?bid=2921361&fileBranchId=3140836#l36)